finality-aleph/src/sync/service.rs

```diff
@@ -313,6 +313,15 @@ where
         }
     }
 
+    fn process_equivocation_proofs(&self, proofs: Vec<V::EquivocationProof>) {
+        for proof in proofs {
+            warn!(target: LOG_TARGET, "Equivocation detected: {proof}");
+            if proof.are_we_equivocating() {
+                panic!("We are equivocating, which is ILLEGAL - shutting down the node. This is probably caused by running two instances of the node with the same set of credentials. Make sure that you are running ONLY ONE instance of the node. If the problem persists, contact the Aleph Zero developers on Discord.");
```

Comment:
we really should have just a page with a contact info and just past a link here -.-

Reply:
I guess that we might create a task for this? Although I've talked with @DamianStraszak and he says this should be fine.

Reply:
I guess it's fine, it's informative and practical. I don't know if we have any page with such contact information.

Reply:
A task would be also about creating such a page. For this pr here its fine for now